app/src/main/res/layout/profile_input_view.xml

```diff
@@ -1,28 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout>
-<LinearLayout
-  xmlns:android="http://schemas.android.com/apk/res/android"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:layout_marginTop="32dp"
-  android:layout_marginStart="32dp"
-  android:layout_marginEnd="32dp"
-  android:orientation="vertical">
-  <TextView
-    android:id="@+id/label_text"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:textColor="@color/black" />
-  <EditText
-    android:id="@+id/input"
-    android:background="@drawable/edit_text_black_border"
+
+  <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginTop="8dp"/>
-  <TextView
-    android:id="@+id/error_text"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:textColor="@color/red"/>
-</LinearLayout>
+    android:layout_marginStart="28dp"
+    android:layout_marginTop="32dp"
+    android:layout_marginEnd="28dp"
+    android:orientation="vertical">
+
+    <TextView
+      android:id="@+id/label_text"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="4dp"
+      android:fontFamily="sans-serif"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="12sp" />
```

Comment:
As you needs only one edit text it would be better you avoid modification for just one edit-text here... This is also a common file used in other places as well with different text-size.